engine-vuex/src/store.ts

```diff
@@ -348,6 +348,20 @@ export class WWTEngineVuexModule extends VuexModule implements WWTEngineVuexStat
     Vue.$wwt.inst.ctl.zoom(factor);
   }
 
+  @Mutation
+  move(obj: { x: number; y: number }): void {
+    if (Vue.$wwt.inst === null)
+      throw new Error('cannot move without linking to WWTInstance');
+    Vue.$wwt.inst.ctl.move(obj.x, obj.y);
```

Comment:
I think this PR is missing a change to `engine/src/index.d.ts` to expose the `move` and `_tilt` methods?

Note that `_tilt` is so named because it's marked as `private` in the C# — and it would actually be somewhat undesirable to make it public, because (e.g.) pywwt relies on the existing, underscore-prefixed name. The joys of maintaining API compatibility.

Reply:
Yep! I will add in the definitions to the .d.ts files.